CardanoSharp.Wallet/TransactionBuilding/ABuilder.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CardanoSharp.Wallet.TransactionBuilding
+{
+    public abstract class ABuilder<T>
```

Comment:
Maybe a `ACollectionBuilder<T>` that provides a `BuildAll()` method could reduce the `Build`-Tails
Or a set of `ACompositeBuilder<T1,T2,T3,...>` with two, three, four, five generic types...

Reply:
yeah we can play around with different mechanics to help clean up the syntax